fivetran_provider_async/operators/fivetran.py

```diff
@@ -0,0 +1,64 @@
+from typing import Any, Dict, Optional
```

Comment:
Same comment as 
https://github.com/astronomer/airflow-provider-fivetran-async/pull/8/files#r1034764486

Reply:
sorry which comment are you referring to here? The link is redirecting to the whole PR

Reply:
it was about moving `fivetran_provider_async/operators/fivetran.py` to `fivetran_provider_async/operators.py`